libraries/Microsoft.Bot.Builder.Dialogs/Memory/Scopes/LGMemoryScope.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Dialogs.Memory.Scopes
+{
+    /// <summary>
+    /// DialogMemoryScope maps "lg" -> dc.Parent?.ActiveDialog.State ?? ActiveDialog.State.
+    /// </summary>
+    public class LGMemoryScope : MemoryScope
```

Comment:
> public class LGMemoryScope : MemoryScope [](http://example.com/codeflow?start=3&length=41)

This entire memory scope shouold not exist.  It is a clone of the existing DialogMemoryScope.cs with a different name.  ?!?!

Reply:
This is due to a bad merge, which i will correct